src/test/python/complete_winml_tutorial.py

```diff
@@ -80,9 +80,9 @@ def print_results(results, labels):
     topProbabilities = [0.0 for x in range(3)]
     topProbabilityLabelIndexes = [0 for x in range(3)]
 
-    for i in range(results.Size):
+    for i in range(results.size):
         for j in range(3):
-            result = results.GetAt(i)
+            result = results.get_at(i)
```

Comment:
Can we use regular iteration here? (`for i, result in enumerate(results):` if we need to keep `i` around)

Reply:
probably. Thanks for the suggestion

Reply:
yup. New code will be:

``` python
    for i, result in enumerate(results):
        for j in range(3):
            if result > topProbabilities[j]:
                topProbabilityLabelIndexes[j] = i
                topProbabilities[j] = result
                break
```